packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,73 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+const NEWLINE_REGEX = /\r\n|[\n\v\f\r\x85\u2028\u2029]/;
+const CONTEXT_LINES = 2;
+
+function padLeft(text, length) {
+  return (' '.repeat(length) + text).slice(-length);
+}
+
+function codeContext(node, source, color) {
+  const lines = source.split(NEWLINE_REGEX);
+
+  const {start, end} = node.node.loc;
+  const startLine = start.line - 1;
+  const startColumn = start.column;
+  const endLine = end.line - 1;
+  const endColumn = end.column;
+  const contextStart = Math.max(0, startLine - CONTEXT_LINES);
+  const contextEnd = Math.min(lines.length, endLine + CONTEXT_LINES);
+
+  let result = '';
+  for (let i = contextStart; i <= contextEnd; i++) {
```

Comment:
I didn't change this to map since we still need access to the index, and I think this way is a bit cleaner than: 
```js
return lines.slice(contextStart, contextEnd + 1).map((line, index) => {
  const index = index + contextStart;
  // do stuff
  return new_line;
}).join('\n');
```